docs/data/osdf/install-cache-rpm.md

```diff
@@ -0,0 +1,125 @@
+* __Network ports:__ The cache service requires the following ports open:
+  * Inbound TCP port 8443 for file access via the HTTP(S) and XRoot protocols.
+  * (Optional) Inbound TCP port cache for access to the web interface for monitoring and configuration;
+    if enabled, this should be restricted to the LAN or management network.
+* __Hardware requirements:__ We recommend that an origin has at least 1Gbps connectivity and 12GB of RAM.
+  We suggest that several gigabytes of local disk space be available for log files,
+  although some logging verbosity can be reduced.
+
+As with all OSG software installations, there are some one-time steps to prepare in advance:
+
+* Obtain root access to the host
+* Prepare [the required Yum repositories](../../common/yum.md)
+
+
+!!! note "OSG 23"
+    In OSG 23, the Pelican-based OSDF RPMs are only available in the "osg-upcoming" repositories.
+
+!!! note "Host certificates"
+    Caches should use a CA that is accepted by major browsers and operating systems,
+    such as InCommon RSA or [Let's Encrypt](../../security/host-certs/lets-encrypt).
+    IGTF certs are not recommended because clients are not configured to accept them by default.
+    Note that you will need the full certificate chain, not just the certificate.
+
+    The following locations should be used (note that they are in separate directories):
+    
+    * **Host Certificate Chain**: `/etc/pki/tls/certs/pelican.crt`
+    * **Host Key**: `/etc/pki/tls/private/pelican.key`
+
+
+Installing the Cache
+--------------------
+
+The cache service is provided by the `osdf-cache` RPM.
+Install it using the following command:
+
+
+```console
+root@host # yum install --enablerepo=osg-upcoming osdf-cache
+```
+
+
+Configuring the Cache Server
+----------------------------
+
+Configuration for a Pelican-based OSDF Cache is located in `/etc/pelican/osdf-cache.yaml`.
+
+You must configure the following:
+```
+XRootD:
+  Sitename: <RESOURCE NAME REGISTERED WITH OSG>
+Cache:
+  DataLocation: "<TOP OF CACHE DIRECTORY>"
+```
+
+If you are using a separate partition for the cached data, which is strongly recommended,
+then use the mount point of the cache partition as `Cache.DataLocation`.
+
+
+Preparing for Initial Startup
+-----------------------------
+
+1.  The cache identifies itself to the federation via public key authentication;
+before starting the cache for the first time, it is recommended to generate a keypair.
+
+    :::command
+    root@host$ cd /etc/pelican
+    root@host$ pelican generate keygen
+
+
+    The newly created files, `issuer.jwk` and `issuer-pub.jwks` are the private and public keys, respectively.
+    **Save these files**; if you lose them, your cache will need to be re-approved.
+
+1.  Contact OSG Staff and let them know that you are about to start your cache,
+    and what the hostname of the cache is.
+    OSG Staff will need to approve the cache's registration.
```

Comment:
I think this section + topology registration should be combined under a single registration section. The fact that we have two different places to register in feels like an implementation detail that users don't necessarily need to be exposed to

Reply:
It would be nice to insulate them from that but there's basically an ordering issue:

- they need to register in Topology before they can configure XRootD.Sitename in their config
- they need to start the cache before we can click the 'approve' button

it's not so much that there are two types of registration, but that cache setup is basically sandwiched between the two types of registration which is why I don't think I can combine the sections.